integr8ly/integreatly-operator
# MGDAPI-5371 Test Case updates for RHOAM v1.33.0

# Issue link


https://issues.redhat.com/browse/MGDAPI-5371

# What

Standard target updates based on what was tested in RHOAM v1.33.0
Minor updates to test cases
Removal of the three customfields. Reason is that after Jira upgrade these cannot be set on ticket creation, only afterwards.
Skip of E05 test since it is failing

# Verification steps




Eye review should suffice. You can run `LOCAL=false INSTALLATION_TYPE=managed-api TEST="E05" make test/e2e/single` if you have cluster (it should pass). And you can run E05 in cluster (using this guide: https://gist.github.com/psturc/87705b3352a1182ba23acc6e92ecd2d8). It should be skipped. However, I have done so. I also tried to run E05 via external image and it passed too. So the issue with E05 is only present if running in cluster.
